.github/workflows/release.yml

```diff
@@ -1,16 +1,32 @@
-name: release
+name: Release
 
 on:
-  push:
-    tags:
-      - "v*"
+  workflow_dispatch:
+    inputs:
+      release:
+        description: "The New Release"
+        required: true
+
+env:
+  RELEASE: ${{ github.event.inputs.release }}
 
 permissions:
   contents: write
 
 jobs:
   release:
     runs-on: ubuntu-latest
+
     steps:
       - uses: actions/checkout@v2
-      - uses: cli/gh-extension-precompile@v1
+
+      - name: Setup Go
+        uses: actions/setup-go@v2.1.4
+        with:
+          go-version: "1.17.3"
+
+      - name: Release
+        run: |
+          echo ${{ secrets.ACCESS_TOKEN }} | gh auth login --with-token
```

Comment:
How does the gh-extension-precompile action work without having an access token?
I did see this in their `action.yml`: `GITHUB_TOKEN: ${{ github.token }}`.
Can we use the same thing?

Reply:
ok I will test it, but it's better to use this command `echo ${{ secrets.ACCESS_TOKEN }} | gh auth login --with-token` for more functionality

Reply:
See https://cli.github.com/manual/gh_help_environment

Reply:
done, I used `${{ github.token }}` and it's worked

Reply:
now you can merge it